src/Base/Exception.h

```diff
@@ -25,8 +25,7 @@
 #ifndef BASE_EXCEPTION_H
 #define BASE_EXCEPTION_H
 
-#include <csignal>
-#include <string>
+#include <sstream>

```

Comment:
This causes the expected build failure:
```
../../../src/Base/Exception.h:744:22: error: field 'new_action' has incomplete type 'Base::sigaction'
  744 |     struct sigaction new_action, old_action;
      |                      ^~~~~~~~~~
```

Reply:
> Why do you handle Exception.h again? It was changed just a week ago.

I thought I could get rid of csignal and move it to the files that actually need it. And my compiler told me that only App/Application needs this. Therefore the PR was the attempt in doing so. obviously this is not doable, so I will retract the PR.

After the removal of csignal the compiler told about stringstream, and since in my 
<sstream>, <string> is loaded, I made this change.

This was my final attempt for Base.